### PR TITLE
Jasmeen/add config imgs 416

### PR DESCRIPTION
**Pull request by KAUR1984**
Fixes the issue #416.

**_Tasks Completed:_** 

- [x] Added the ACCESS-AM image.
- [x] Adjusted the ACCESS-CM image.
- [x] Adjusted the ACCESS-OM image.
- [x] Adjusted the ACCESS-ESM image.

> These images are very much down to changes and improvements in the future. Just adding them now for the soft release and perhaps, we could get some feedback for potential improvements.

### CODE REVIEW

File: docs/css/access-nri.css [hunk 1]
```diff
@@ -667,6 +667,11 @@ pre>code {
   box-sizing: border-box;
 }
 
+/* Add padding to image container */
+.with-padding {
+  padding: 0.5em;
+}
+
```
Comment (KAUR1984) [inline docs/css/access-nri.css:674]: Added additional class for a little padding around the image. Might need to check if this is ok for the overall CSS code quality. @atteggiani :)

File: docs/models/configurations/access-am.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 <!-- {% include "call_contribute.md" %} -->
 
+<img src="../../../assets/model-config-logos/access-am-config.png" alt="ACCESS AM model" class="image-background center-img with-border with-padding"></img>
+
```
Comment (KAUR1984) [inline docs/models/configurations/access-am.md:6]: Used the above `with-padding` CSS class here.

File: docs/models/configurations/access-cm.md [hunk 3]
```diff
@@ -1,12 +1,10 @@
 # <div class="center-icons"> ACCESS-CM  </div>
 
-<div style="text-align: justify">
 ACCESS-CM2 (ACCESS Coupled Model 2) is a global fully-coupled climate model that includes the atmosphere, ocean and sea-ice components, and produces physical climate simulations. ACCESS-CM2 is one of the two models run by the Australian climate community for the 
 <a href = "https://www.wcrp-climate.org/wgcm-cmip" target="_blank"> Coupled Model Intercomparison Project, CMIP</a>.
 <br>
-</div>
 
-![ACCESS-CM2 Logo](../../assets/component-logos/access-cm2-1.9.png){width=100%}
+<img src="../../../assets/model-config-logos/access-cm2-config.png" alt="ACCESS CM model" class="image-background center-img with-border with-padding"></img>
```
Comment (KAUR1984) [inline docs/models/configurations/access-cm.md:7]: Used the above `with-padding` CSS class here.

File: docs/models/configurations/access-esm.md [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 <!-- {% include "call_contribute.md" %} -->
 
+<img src="../../../assets/model-config-logos/access-esm-config.png" alt="ACCESS CM model" class="image-background center-img with-border with-padding"></img>
```
Comment (KAUR1984) [inline docs/models/configurations/access-esm.md:5]: Used the above `with-padding` CSS class here.

File: docs/models/configurations/access-om.md [hunk 5]
```diff
@@ -3,6 +3,8 @@
 
 <!-- {% include "call_contribute.md" %} -->
 
+<img src="../../../assets/model-config-logos/access-om2-config.png" alt="ACCESS CM model" class="image-background center-img with-border with-padding"></img>
```
Comment (KAUR1984) [inline docs/models/configurations/access-om.md:6]: Used the above `with-padding` CSS class here.

File: docs/models/configurations/index.md [hunk 6]
```diff
@@ -3,7 +3,7 @@
 <div class="card-container" style="flex-direction: column">
     <a href="access-cm/" class="justified rectangular-card configuration-card default-text-color">
             <div class="rectangular-card-image-container">
-                <img src="../../assets/model-config-logos/access-cm2-config-4.png" class="image-background img-cover"></img> 
+                <img src="../../assets/model-config-logos/access-cm2-config.png" class="image-background img-cover"></img>
```
Comment (KAUR1984) [inline docs/models/configurations/index.md:6]: Fixed the `Not found` image error due to incorrect address.

File: docs/models/index.md [hunk 7]
```diff
@@ -10,7 +10,7 @@ ACCESS is a family of related computer models that are able to represent differe
 <div class="card-container" style="flex-direction: column">
     <a href="configurations/access-cm/" class="justified rectangular-card configuration-card default-text-color">
             <div class="rectangular-card-image-container">
-                <img src="../assets/model-config-logos/access-cm2-config-4.png" class="image-background img-cover"></img> 
+                <img src="../assets/model-config-logos/access-cm2-config.png" class="image-background img-cover"></img>
```
Comment (KAUR1984) [inline docs/models/index.md:13]: Fixed the `Not found` image error due to incorrect address.

File: docs/models/model_components/aerosols_atmospheric_chemistry.md [hunk 8]
```diff
@@ -4,7 +4,7 @@
 
 <!-- ![Atmospheric Component Logo](../../assets/component-logos/components-without-titles/ACCESS%20icon%20AEROSOLS.png){align=right width=40%} -->
 
-<img src = "../../../assets/component-logos/component-maps/aerosol-chemistry-component-map.png" class="white-img-bg"></img>
+<img src = "../../../assets/component-logos/component-maps/aerosol-chemistry-component-map.png" class="image-background"></img>
```
Comment (KAUR1984) [inline docs/models/model_components/aerosols_atmospheric_chemistry.md:7]: Replaced the `white-img-bg` class which does not exist anymore, with the new one.

File: docs/models/model_components/atmosphere.md [hunk 9]
```diff
@@ -4,7 +4,7 @@
 
 <!-- ![Atmosphere Component Logo](../../assets/component-logos/components-without-titles/ACCESS icon ATMOSPHERE.png){align=right width=40%} -->
 
-<img src = "../../../assets/component-logos/component-maps/atmosphere-component-map.png" class="white-img-bg"></img>
+<img src = "../../../assets/component-logos/component-maps/atmosphere-component-map.png" class="image-background"></img>
```
Comment (KAUR1984) [inline docs/models/model_components/atmosphere.md:7]: Replaced the `white-img-bg` class which does not exist anymore, with the new one.

File: docs/models/model_components/bgc_land.md [hunk 10]
```diff
@@ -5,7 +5,7 @@
 
 <!-- ![BGC Component Logo](../../assets/component-logos/components-without-titles/ACCESS icon BGC LAND.png){align=right width=40%} -->
 
-<img src = "../../../assets/component-logos/component-maps/bgc-land-component-map.png" class="white-img-bg"></img>
+<img src = "../../../assets/component-logos/component-maps/bgc-land-component-map.png" class="image-background"></img>
```
Comment (KAUR1984) [inline docs/models/model_components/bgc_land.md:8]: Replaced the `white-img-bg` class which does not exist anymore, with the new one.

File: docs/models/model_components/bgc_ocean.md [hunk 11]
```diff
@@ -4,7 +4,7 @@
 
 <!-- ![BGC Ocean Component Logo](../../assets/component-logos/components-without-titles/ACCESS icon BGC OCEAN.png){align=right width=40%} -->
 
-<img src = "../../../assets/component-logos/component-maps/bgc-ocean-component-map.png" class="white-img-bg"></img>
+<img src = "../../../assets/component-logos/component-maps/bgc-ocean-component-map.png" class="image-background"></img>
```
Comment (KAUR1984) [inline docs/models/model_components/bgc_ocean.md:7]: Replaced the `white-img-bg` class which does not exist anymore, with the new one.

File: docs/models/model_components/coupler.md [hunk 12]
```diff
@@ -2,7 +2,7 @@
 
 <!-- ![Coupler Component Logo](../../assets/component-logos/components-without-titles/ACCESS icon COUPLER.png){align=right width=40%} -->
 
-<img src = "../../../assets/component-logos/component-maps/coupler-component-map.png" class="white-img-bg"></img>
+<img src = "../../../assets/component-logos/component-maps/coupler-component-map.png" class="image-background"></img>
```
Comment (KAUR1984) [inline docs/models/model_components/coupler.md:5]: Replaced the `white-img-bg` class which does not exist anymore, with the new one.

File: docs/models/model_components/land.md [hunk 13]
```diff
@@ -4,7 +4,7 @@
 
 <!-- ![Land Component Logo](../../assets/component-logos/components-without-titles/ACCESS icon LAND SURFACE.png){align=right width=40%} -->
 
-<img src = "../../../assets/component-logos/component-maps/land-component-map.png" class="white-img-bg"></img>
+<img src = "../../../assets/component-logos/component-maps/land-component-map.png" class="image-background"></img>
```
Comment (KAUR1984) [inline docs/models/model_components/land.md:7]: Replaced the `white-img-bg` class which does not exist anymore, with the new one.

**Comment by heidinett** [comment]
Thanks Jasmeen for adding the images in the different model configurations. Looks good! 
For the image in ACCESS-AM, is it possible to reduced the font size of "ACCESS-AM" heading to match the same size of the other configuration headings?

**Comment by atteggiani** [comment]
Thanak you Jasmeen!! 
Everything looks very good :) 

The only two comments that I have, related to what @heidinett said are:
- To also make ACCESS-OM2 the same fontsize as the other ones. (Basically use the same fontsize for all the configurations);
- To change the ACCESS-CM2 component name from "land surface" to "land", as in the other configurations that use it (ESM and AM)

👍

**Comment by KAUR1984** [comment]
Thanks so much for the suggestions @heidinett @atteggiani ! Very happy to make these tweaks - https://github.com/ACCESS-Hive/access-hive.github.io/issues/435. Would it be ok if I create a separate pr for that one?

**Comment by heidinett** [comment]
Sure, no problem! Thanks Jasmeen 🙂


Heidi Nettelbeck                                         [cid:803ea55b-dc4a-4522-b947-0f9f4b7245f4]

Team Leader, Atmospheric Modelling Infrastructure

Australian Earth System Simulator (ACCESS-NRI)

T: +61 02 6125 4959 | E: ***@***.******@***.***> | W: access-nri.org.au<https://www.access-nri.org.au/> |

We at ACCESS-NRI acknowledge the Traditional Owners of the land on which our research infrastructure and community operate across Australia and pay our respects

to Elders past and present. We recognise the thousands of years of accumulated knowledge and deep connection they have with all the Earth systems we simulate.


________________________________

**Comment by atteggiani** [comment]
> Thanks so much for the suggestions @heidinett @atteggiani ! Very happy to make these tweaks - #435. Would it be ok if I create a separate pr for that one?

Why do you want to create a separate PR? 
Wouldn't be easier just to keep this open so you can push to the same branch and this gets updated automatically?

**Comment by KAUR1984** [comment]
This pr was mainly aimed at adding same size images for all model configurations and changing the split image for CM2. Because of my newbie photoshop skills, I am still experimenting with different designs for images and ensuring consistent labelling and sizes. This could take some time, and I am a bit hesitant to keep this pr open till then. Have created a separate issue for the suggestions above to make it easy for us to track progress. Let me know if that sounds good @atteggiani :) .

**Comment by atteggiani** [comment]
> This pr was mainly aimed at adding same size images for all model configurations and changing the split image for CM2. Because of my newbie photoshop skills, I am still experimenting with different designs for images and ensuring consistent labelling and sizes. This could take some time, and I am a bit hesitant to keep this pr open till then. Have created a separate issue for the suggestions above to make it easy for us to track progress. Let me know if that sounds good @atteggiani :) .
> 
> Very happy to work on the same branch too if we decide, but would suggest we close this pr without merging into `development` since it's best if we avoid open pr's which are still in progress. :)

That's alright then! 👍

**Comment by KAUR1984** [comment]
Great! Thanks @atteggiani !

**Pull request merged**